about.html

```diff
@@ -4,24 +4,24 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>capstone</title>
+  <title>SCUT Alumni Association</title>
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" />
-  <link rel="stylesheet" href="styles/about.css">
+  <link rel="stylesheet" href="about.css">
 </head>
 <body>
 <div class="startNav">
   <ul class="startNavUl">
     <li class="startNavLi">
-      <a href="https://web.facebook.com/alaminrakib.tonoy"><i class="social fab fa-facebook-f"></i></a>
+      <a href="#"><i class="social fab fa-facebook-f"></i></a>
     </li>
-    <li class="startNavLi"><a href="https://twitter.com/elonmusk"><i class="social fab fa-twitter"></i></a></li>
+    <li class="startNavLi"><a href="https://twitter.com/msd_811"><i class="social fab fa-twitter"></i></a></li>
     <li class="startNavLi"><a href="#" class="startNavLink">English</a></li>
     <li class="startNavLi"><a href="#" class="startNavLink">Logout</a></li>
   </ul>
 </div>
 <header class="header">
-  <a href="index.html"><img class="logo" src="image/icons/qawaliLogo.png" alt="logo" /></a>
+  <a href="index.html"><img class="logo" src="image/icons/capstone1.png" alt="logo" /></a>
   <nav class="nav">
     <ul class="navUl">
       <li class="navLi"><a href="index.html" class="navLink">Home</a></li>
```

Comment:
- [ ] (OPTIONAL) Great job so far 👍 Kindly make sure that your About page navbar looks exactly like the Home page navbar. Right now the navbar of the About page has fewer links and it is not styled well. Kindly correct this so as to improve your site's UI 💯 
- About navbar:
![Annotation 2022-04-23 052910](https://user-images.githubusercontent.com/85113567/164877730-2b535cd0-eb09-42eb-94e8-4270a65f9bf3.jpg)
- Home navbar:
![Annotation 2022-04-23 052935](https://user-images.githubusercontent.com/85113567/164877738-762b6b1e-6d15-4730-9ed7-4669c67a64ea.jpg)

Reply:
Thank you for your suggestion. I didn't notice it. I'll make the changes 🙏🏻.